Gruntfile.js

```diff
@@ -59,18 +59,24 @@ module.exports = function(grunt) {
 
         var currentApp = queuedTests[0],
             currentTestPath = BASE_URL + currentApp,
-            testText = "Running Test '" + currentApp + "'";
+            testText = "Running Test '" + currentApp + "'",
+            runner_options = {
+                file: currentTestPath,
+                visible: false,
+                timeout: 120000,
+                reporter: 'dot',
+            };
+
+        // For running in docker/travis
+        if (process.env.PUPPETEER_SKIP_CHROMIUM_DOWNLOAD) {
```

Comment:
What's puppeteer?

Reply:
`puppeteer` is google's response to `selenium` for running ui tests in chrome I believe, but `mocha-chrome-headless` requires it as a dependency.